apps/trading/pages/portfolio/deposit/deposit-container.tsx

```diff
@@ -0,0 +1,60 @@
+import type { EthereumConfig } from '../../../components/web3-container/web3-container';
+import { gql } from '@apollo/client';
+import { PageQueryContainer } from '../../../components/page-query-container';
+import type { DepositPage } from './__generated__/DepositPage';
+import { DepositManager } from '@vegaprotocol/deposits';
+import { t } from '@vegaprotocol/react-helpers';
+import { Splash } from '@vegaprotocol/ui-toolkit';
+
+const DEPOSIT_PAGE_QUERY = gql`
+  query DepositPage {
+    assets {
+      id
+      symbol
+      name
+      decimals
+      source {
+        ... on ERC20 {
+          contractAddress
+        }
+      }
+    }
+  }
+`;
+
+interface DepositContainerProps {
+  ethereumConfig: EthereumConfig;
+  assetId?: string;
+}
+
+/**
+ *  Fetches data required for the Deposit page
+ */
+export const DepositContainer = ({
+  ethereumConfig,
+  assetId,
+}: DepositContainerProps) => {
+  return (
+    <PageQueryContainer<DepositPage> query={DEPOSIT_PAGE_QUERY}>
+      {(data) => {
+        if (!data.assets?.length) {
+          return (
+            <Splash>
+              <p>{t('No assets on this network')}</p>
+            </Splash>
+          );
+        }
+
+        return (
+          <DepositManager
+            bridgeAddress={ethereumConfig.collateral_bridge_contract.address}
+            requiredConfirmations={ethereumConfig.confirmations}
+            // @ts-ignore TS not inferring on union type for contract address
+            assets={data.assets.filter((a) => a.source.__typename === 'ERC20')}
```

Comment:
nit[ick (non-blocking): id there a better way to do this for example an `as` or similar so we can avoid the TS ignore